### PR TITLE
Add RPC mock server for integration tests again, adjust tests

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

**Comment by vimukthi-git** [comment]
@philipstanislaus are you waiting for my review here?

**Comment by philipstanislaus** [comment]
Yes, thanks @vimukthi-git . Will fix the tests shortly

**Comment by philipstanislaus** [comment]
Fixed @vimukthi-git :)

File: rpcmocksrv/server.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+	*rpc.Server
+	// Host consists of hostname and port
+	Host string
+	// URL consists of protocol, hostname and port
+	URL string
+}
+
+// Following 4 methods are not go routine safe
+
```
Comment (vimukthi-git) [inline rpcmocksrv/server.go]: do we need to keep these commented out things?

File: rpcmocksrv/server_test.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package rpcmocksrv
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/stretchr/testify/assert"
+)
+
+// const (
```
Comment (vimukthi-git) [inline rpcmocksrv/server_test.go]: same here

**Comment by philipstanislaus** [comment]
These were things that still needed to be migrated – but I'll keep track of them elsewhere. Comments removed @vimukthi-git

**Pull request merged**